src/filter_tree/filter_tree.h

```diff
@@ -40,21 +55,34 @@ typedef struct {
 /* All nodes within the filter tree are of type FT_FilterNode. */
 struct FT_FilterNode {
 	union {
+        FT_ExpressionNode exp;
```

Comment:
Spaces here instead of tabs (also true for lines 33 and 22)

Reply:
?

Reply:
The added lines in this file use 8 space characters instead of 2 tab characters for whitespace. Were you working on a machine without the autoformatter or something?